website/azurerm.erb

```diff
@@ -445,6 +445,10 @@
 				<li<%= sidebar_current("docs-azurerm-datasource-virtual-network-gateway-connection") %>>
 					<a href="/docs/providers/azurerm/d/virtual_network_gateway_connection.html">azurerm_virtual_network_gateway_connection</a>
 				</li>
+
+				<li<%= sidebar_current("docs-azurerm-datasource-network-virtual-hub") %>>
+                    <a href="/docs/providers/azurerm/d/virtual_hub.html">azurerm_virtual_hub</a>
```

Comment:
The formatting here isn't exactly lined up.

Reply:
fixed